src/pytest_mysql/factories/process.py

```diff
@@ -17,12 +17,12 @@
 # along with pytest-mysql.  If not, see <http://www.gnu.org/licenses/>.
 """Process fixture factory for MySQL database."""
 
-import os
+import os, py
 from warnings import warn
 
 import pytest
 from _pytest.fixtures import FixtureRequest
-from _pytest.tmpdir import TempdirFactory
+from _pytest.tmpdir import TempPathFactory
```

Comment:
`from pytest import FixtureRequest, TempPathFactory`

Reply:
I changed it!